TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift

```diff
@@ -40,8 +56,13 @@ class PushFactory {
 }
 
 extension PushFactory: PushFactoryProtocol {
-  func createFactor(withAccessToken accessToken: String, friendlyName: String, pushToken: String, serviceSid: String,
-                    identity: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
+  func createFactor(withAccessToken accessToken: String,
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/0c8d089e020078c52d469dcc5b1c3a5eb36c4ee1/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift#L59"></span>Function body should span 40 lines or less excluding comments and whitespace: currently spans 43 lines<br />
<code>function_body_length</code> <code>PushFactory.swift:59</code></td>
    </tr>
  </tbody>
</table>